glide.yaml

```diff
@@ -47,6 +47,8 @@ import:
   version: master
 - package: github.com/openshift/service-serving-cert-signer
   version: release-3.10
+- package: github.com/openshift/source-to-image
+  version: release-3.10
```

Comment:
Pin here is to exclude the `--as-dockerfile` feature @bparees recently added. This is still experimental and we don't want this in Origin until 3.11